### ISSUE TITLE
remove string WA for material.io Date picker

### ISSUE THREAD

**Issue by bekuno** [cgeo/cgeo#14431]
### Tell us your idea!

Remove the temporarily string in `strings.xml`, as the error is solved in the meantime, the issue  https://github.com/material-components/material-components-android/issues/1346 is fixed.

~~~
<!-- This entry shall solve a bug with material.io Date picker, see https://github.com/material-components/material-components-android/issues/1346 -->
<string name="mtrl_picker_confirm" tools:override="true">OK</string>
~~~

@eddiemuc OK?

**Comment by eddiemuc** [comment]
Ok

**Comment by moving-bits** [comment]
Implemented with #14432
